apps/scan/backend/src/scanners/custom/app_scan_double_sheets.test.ts

```diff
@@ -90,10 +93,9 @@ test('insert second ballot while first ballot is accepting', async () => {
         DELAY_ACCEPTED_READY_FOR_NEXT_BALLOT: 1000,
         DELAY_ACCEPTED_RESET_TO_NO_PAPER: 2000,
       },
-      interpret: mockInterpret(interpretation),
```

Comment:
The tldr here is that you can only use mocked interpretation (as opposed to real interpretation) if you don't need to export CVRs in your test. With continuous export no longer feature flag gated, the number of tests that export CVRs and can't use mock interpretation has increased.

How I figured this out:

https://github.com/votingworks/vxsuite/blob/1483679cbc86ed63b93da1103b291a323a4c4a78/apps/scan/backend/test/helpers/shared_helpers.ts#L103-L117